.github/ISSUE_TEMPLATE/coverage_impact_report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Coverage Impact Report
+labels: type:Coverage
+about: File a coverage report to improve code coverage in ITK
+---
+
+<!-- The text within this markup is a comment, and is intended to provide
+guidelines to open an issue for the ITK repository. This text will not
+be part of the issue. -->
+
+
+<!-- Before submitting an issue, please check that your issue has not been
+already filed. -->
+
+### Description
+
+<!-- Description of the impact of the current coverage in a given code area, and
+how an increased coverage would benefit the toolkit. -->
+
+### Steps to Reproduce
```

Comment:
I don't think we need many of these section for the coverage report -- they will largely be unused.

Reply:
Done in 72eae4b.